lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -34,7 +35,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToKbString(bytes) {
-    return Math.round(bytes / KB_IN_BYTES).toLocaleString() + ' KB';
+    return Util.formatBytesToKB(bytes);
```

Comment:
Ok, is it filtered on the audit level or somewhere further? If on audit level, then I can't see any such condition in this one so it returns everything. I found this example using some fake data in a unit test. I'll add it so you'll know during future fixing that this kind of result shouldn't be there.